const aa = 1111
async function o() {
    let rst = await new Promise((resolve, reject) => {
        reject('抛出错误');
    });
    let b = 2;
}
